content/using/install.md

```diff
@@ -148,217 +108,133 @@ use |ota %disable or |ota ~sponsor %kids to reset it
 
-We have a guide for [hosting your ship on DigitalOcean](@/using/operations/hosting.md) which we've found works well, but any cloud hosting service should work.
+### Run the boot command {#the-dojo}
 
-### Boot your planet
+Navigate to your `urbit` directory.
 
-You'll be booting your ship with the keyfile that you downloaded from Bridge.
-
-#### Step 1: Find the path to your keyfile
-
-Find the absolute path to the keyfile that you downloaded from Bridge. Copy it.
-
-#### Step 2: Run the boot command
-
-Enter your `urbit` directory.
-
-Once you're inside, run the command below, except with `sampel-palnet` replaced by the name of your
-Urbit identity, and `path/to/my-planet.key` replaced with the path to your keyfile:
-
-```sh
-./urbit -w sampel-palnet -k path/to/my-planet.key
-```
-
-Or, if you'd prefer to copy-paste your key, you can run:
+Open your keyfile (it is just a text file) and copy the key inside.  Paste it into the following command, with `sampel-palnet` replaced by the name of your
+planet.
 
 ```sh
 ./urbit -w sampel-palnet -G rAnDoMkEy
 ```
 
-Either command will create a directory called `sampel-palnet/` and begin building your ship. It may take a few minutes.
-
-When your ship is finished booting, you will see either the `~sampel-palnet:dojo>` or `~sampel-palnet:chat-cli/` prompt. If you're seeing `:chat-cli` press `Ctrl-X` to switch into Dojo.
-
-To shut down your ship, use `Ctrl-D`. To start your ship up again, run the following from your `urbit` directory:
+Or you can copy the keyfile into the current directory and run:
 
 ```sh
-./urbit sampel-palnet
+./urbit -w sampel-palnet -k ./my-planet.key
 ```
 
-Note that `sampel-palnet/` is the path of a folder, which we just created in your `urbit` directory. This folder is called your ship's **pier**.
+Either command will create a directory called `sampel-palnet/` and begin building your planet. It may take a few minutes.
 
-Never boot multiple instances of your ship at the same time. You can prevent this from happening on accident by only ever keeping a single copy of your pier.
+When your planet is finished booting, you will see either the `~sampel-palnet:dojo>` or `~sampel-palnet:chat-cli/` prompt. If you're seeing `:chat-cli` press `Ctrl-X` to switch into Dojo (the Urbit command line).
 
-**Important:** once a key has been used to boot a ship onto the network, it cannot be used to boot that ship again later - doing so will cause communication problems with other ships. For this reason you should **delete the keyfile from your machine once your ship has booted successfully**. If you do use the same key twice, you'll need to conduct a [personal breach](#breaches) to restore your ship to full functionality).
-
-Delete the keyfile with the command below:
+Get your password to log into landscape (the Urbit web interface) by typing:
```

Comment:
You also need ```+code``` to start a comet, so this should probably be mentioned in the comet section above.
"Landscape" is usually capitalized.